Gemfile

```diff
@@ -10,4 +10,5 @@ group :test, :remote_test do
   gem 'jose', '~> 1.1.3'
   gem 'jwe'
   gem 'mechanize'
+  gem 'rexml'
```

Comment:
We have had an issue in which none of the tests can run unless this gem is added to the gemfile. It is the following error:

`LoadError: cannot load such file -- rexml/document`

It doesn;t appear to be related to any code of ours but on the file active_merchant.rb I can add the stack trace if needed.

Reply:
I don't get an error when I comment this gem out of the gemfile and run your tests. Also, as you mention it's already listed in required in the `active_merchant.rb` base file. Can you do me a favor and squash your commits to a single commit and then pull and rebase upstream/master? That's one of the Active Merchant conventions to keep the commit history clean. Several of the CI checks are failing and it might be related to an update that you haven't pulled in.